src/components/SignUpForm.js

```diff
@@ -133,6 +172,34 @@ class SignUpForm extends Component {
           error={fieldErrors.password}
           onChange={this.handleChange} />
 
+        <Checkbox
```

Comment:
This checkbox should have margin below so it doesnt abut the bottom of the container.